app/views/themes/institutional_repository/_masthead.html.erb

```diff
@@ -1,25 +1,20 @@
 <% # OVERRIDE: Hyrax v5.0.0rc2 - added the search bar and removed the /login and locale nav menu and moved to the /controls partial for theming %>
 <header aria-label="header" class="top-header">
-  <nav id="masthead" class="navbar navbar-expand-lg navbar-dark bg-dark justify-content-between institutional-repository-nav <%= placement_class %>" role="navigation" aria-label="masthead">
```

Comment:
Removing navbar navbar-expand-lg removed the color behavior for institutional header

Reply:
Fixed!